app/controllers/application_letters_controller.rb

```diff
@@ -57,6 +57,6 @@ def set_application
 
     # Only allow a trusted parameter "white list" through.
     def application_params
-      params.require(:application_letter).permit(:motivation, :event_id, :status)
+      params.require(:application_letter).permit(:motivation, :user_id, :event_id, :status)
```

Comment:
I think this was removed a couple PRs earlier. I don't know your code that well, I just want to make sure this is not a merge error. Please give it a second look 🙂